src/Akka.HealthCheck/AkkaHealthCheck.cs

```diff
@@ -44,14 +45,22 @@ public sealed class AkkaHealthCheck : IExtension
         public AkkaHealthCheck(HealthCheckSettings settings, ExtendedActorSystem system)
         {
             Settings = settings;
+            system.Log.Info("Liveness Prove Provider: {0}", Settings.LivenessProbeProvider);
```

Comment:
Looks good, but need to add that `log-info` setting here and check to see if that's on before logging all of these